pkg/scripts/download_test.go

```diff
@@ -30,8 +30,8 @@ func (f *FakeHTTPGet) get(url string) (*http.Response, error) {
 	}, f.err
 }
 
-func getHTTPReader() (*HttpURLReader, *FakeHTTPGet) {
-	sr := &HttpURLReader{}
+func getHTTPReader() (*HTTPURLReader, *FakeHTTPGet) {
```

Comment:
I renamed `NewHTTPURLReader` because that has a special semantic value: `godoc` understands that that is a "constructor" for the type and places the function under the type.

See for instance https://golang.org/pkg/bytes/#Reader. It starts with the type definition, then the `NewReader` function, then all methods of the `Reader` type.

This test function doesn't carry the same value, I didn't intend to go rewrite huge portions of the code base in one shot. Perhaps this fits in the "less churn" you noted?